website/content/docs/platform/k8s/injector/installation.mdx

```diff
@@ -65,6 +65,32 @@ The following is required to configure TLS manually:
 
 For more information on configuring manual TLS, see the [Vault Helm cert values](/docs/platform/k8s/helm/configuration#certs).
 
+## Multiple Replicas and TLS
+
+The Vault Agent Injector can be run with multiple replicas if using [Manual
+TLS](#manual-tls), and as of v0.7.0 multiple replicas are also supported with
+[Auto TLS](#auto-tls). The number of replicas is controlled in the Vault Helm
+chart by the [injector.replicas
+value](/docs/platform/k8s/helm/configuration#replicas).
+
+With Auto TLS, a leader-elector sidecar container is deployed with each replica.
```

Comment:
Is it worth explicitly stating that the leader elector sidecar is only deployed if replicas > 1, and the default is 1?

Reply:
Yeah, that's a good point. I think folks should be referring to the injector.replicas link in the previous paragraph for the current default, and I'll add something similar to your other suggestion about multiple replicas here.